plugins/flutter_aepuserprofile/android/gradlew

```diff
@@ -0,0 +1,234 @@
+#!/bin/sh
+
```

Comment:
I don't see these three files in other extensions:  gradlew, gradlew.bat, gradle/wrapper/gradle-wrapper.jar , should we remove them?

Reply:
It should be fine to remove those files with the way we have set up sample app.